server/sonar-server/src/test/java/org/sonar/server/permission/PermissionFinderTest.java

```diff
@@ -60,23 +63,28 @@
   @Mock
   PermissionTemplateDao permissionTemplateDao;
 
-  PermissionFinder finder;
+  PermissionFinder underTest;
 
   @Before
   public void setUp() {
+    DbClient dbClient = mock(DbClient.class);
```

Comment:
It's out of the scope of this PR, but I don't like that a classes using a lof of DAO is not using a real db in its test...